README.md

```diff
@@ -21,21 +21,152 @@ System.config({
+
+    // SCSS
+    'platform/core': path.resolve(__dirname, 'platform/core')
+};
+```
+
+Then, in the webpack configuration you will need to add the following rules:
+
+```javascript
+    module: {
+        rules: [
+            /*
+            * Send all ts files from @nifi-fds through a custom loader that replaces its usage of  templateUrl or styleUrl properties like:
+            *     templateUrl: './confirm-dialog.component.html'
+            *
+            * with NodeJS `require` calls for loading of html files that are subsequently loaded via webpack's html-loader like:
+            *     template: require('./confirm-dialog.component.html')
+            *
+            * And all files with a `.ts` or `.tsx` extension will be handled by `ts-loader`
+            */
+            {
+                test: /\.tsx?$/,
+                include: [
+                    path.resolve(__dirname, 'webapp'),
+                    path.resolve(__dirname, 'platform')
+                ],
+                use: [path.resolve(__dirname, 'angular-url-loader'), 'ts-loader']
+            },
+            {
+            /*
+            * Send all js files from @nifi-fds through a custom loader that replaces its usage of  templateUrl or styleUrl properties like:
+            *     templateUrl: './confirm-dialog.component.html'
+            *
+            * with NodeJS `require` calls for loading of html files that are subsequently loaded via webpack's html-loader like:
+            *     template: require('./confirm-dialog.component.html')
+            *
+            * And all files with a `.js` extension will be handled by `babel-loader`
+            */
+                test: /\.js$/,
+                include: [
+                    path.resolve(__dirname, 'webapp'),
+                    path.resolve(__dirname, 'platform')
+                ],
+                use: [
+                    {
+                        loader: path.resolve(__dirname, 'angular-url-loader')
+                    },
+                    {
+                        loader: 'babel-loader',
+                        options: {
+                            presets: ['@babel/preset-env']
+                        }
+                    }
+                ]
+            },
+            {
+                test: /\.scss$/,
+                use: [
+                        {
+                            // Create CSS files separately
+                            loader: MiniCssExtractPlugin.loader
+                        },
+                        {
+                            // Translate CSS into CommonJS
+                            loader: 'css-loader',
+                            options: {
+                                url: false
+                            }
+                        },
+                        {
+                         // Compile Sass to CSS
+                         loader: 'sass-loader'
+                        }
+                ]
+            }
+            ...
```

Comment:
probably need to add the `html-loader` as well.

Reply:
added